routes/api/category-routes.js

```diff
@@ -0,0 +1,82 @@
+const router = require('express').Router();
+const { Category, Product } = require('../../models');
+
+// The `/api/categories` endpoint
+
+router.get('/', async (req, res) => {
```

Comment:
Although you can write the functions directly in the routes, it is better practice to separate these into another file within a controllers folder. This keeps the routes clean and easy to follow.